1693914802.6059604 [INFO] [launch]: All log files can be found below /home/pi/.ros/log/2023-09-05-11-53-22-598744-ubuntu-1204
1693914802.6071653 [INFO] [launch]: Default logging verbosity is set to INFO
1693914803.7804182 [ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught exception when trying to load file of format [py]: 'LDS_MODEL'
